### PR TITLE
[READY] add featured image to featured post on homepage 

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] make a partial (component) for blog-article-preview
- [x] add it to featured and blog index

Thanks @sn3p for working this out with me! 👍

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
blog-homepage

File: source/blog/_featured.erb [hunk 1]
```diff
@@ -19,20 +12,7 @@ if featured
...
-          <time datetime="<%= featured.date.strftime('%F') %>"><%= I18n.l(featured.date, format: "%e %B %Y", locale: I18n.locale.to_s) %></time>
-        </div>
-      </header>
-
-      <div class="post-body">
-        <%= Nokogiri::HTML(featured.summary(400)).text %>
-        <%= link_to t("blog.read_more"), featured.url, class: "read-more" %>
-      </div>
-    </article>
+    <%= partial "blog/blog-article-preview", locals: {article: featured} %>
```
Comment (fatboypunk) [inline source/blog/_featured.erb]: `locals: { article: featured }`
:dog:

**Comment by fatboypunk** [comment]
Two styling issues, but everything else looks good. Nice improvements 💃

File: source/blog/index.html.erb [hunk 2]
```diff
@@ -10,47 +10,9 @@ pageable: true
...
-                  <div class="author">
-                    <span class="author-name"><%= author_name %></span>
-                    <time datetime="<%= article.date.strftime('%F') %>"><%= I18n.l(article.date, format: "%e %B %Y", locale: I18n.locale.to_s) %></time>
-                  </div>
-                </div>
-              </footer>
-            </article>
-
-          </div>
+          <%= partial "blog/blog-article-preview", locals: {article: article} %>
```
Comment (RonaldDijkstra) [inline source/blog/index.html.erb]: Thx

**Comment by RonaldDijkstra** [comment]
Thnx, done

**Pull request merged**